services/web/client/source/class/osparc/utils/Services.js

```diff
@@ -279,6 +279,15 @@ qx.Class.define("osparc.utils.Services", {
       return null;
     },
 
+    isDeprecated: function(metadata) {
+      if (metadata && "deprecated" in metadata && ![null, undefined].includes(metadata["deprecated"])) {
+        const depTime = new Date(metadata["deprecated"]);
+        const now = new Date();
```

Comment:
Suggestion: Let's always use UTC time, so we dont have to deal with timezone-headches. Not sure if this is anyway assumed here.

Reply:
Already using UTC 👌